ducanhpham0312/zeevision
# feat: incidents tables

### Linked issue

Closes ducanhpham0312/zeevision-private#66
Closes ducanhpham0312/zeevision-private#129

### Description

- Add tables for `/incidents` and `/instances/id` page.
- Regarding issue #129 , the customer has confirmed that when variable's `value` is a json string, it is already prettified, so we don't need another function in the frontend to prettify it. This PR removes that function and use the <pre> tag.
- Update sample query to have prettified json as variable's `value`

### Testing

- Run test bench and the application. Insert the sample query and open pgAdmin
- The IncidentsPage should show 3 incidents that corresponds to 2 instances
- Open either of the instance page, switch to the Incidents tab. There should be either 1 or 2 corresponding incident(s). 
- The error message is exactly what is queried (not being parsed in one line).
- Open `/instances/12` page, the json variable `value` is prettified.
